android/app/src/androidTest/java/com/react/testapp/ManifestTest.kt

```diff
@@ -1,8 +1,8 @@
-package com.react.testapp
+package com.microsoft.reacttestapp
```

Comment:
Do we need to rename the folders as well to match?

Reply:
We need to rename `androidTest/java/com/react/testapp/` and `flipper/java/com/react/testapp/` as well

Reply:
Fixed